package exception.throws_;

public class MainClass {
	public static void main(String[] args) {
		
		
	}
}
